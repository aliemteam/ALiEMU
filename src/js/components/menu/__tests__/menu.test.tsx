import React from 'react';
import renderer from 'react-test-renderer';

import MenuDivider from '../divider';
import Menu from '../index';
import MenuItem from '../item';

it('should render correctly', () => {
    const tree = renderer.create(
        <Menu>
            <MenuDivider title="Testing" />
            <MenuItem>One</MenuItem>
            <MenuItem selected>Two</MenuItem>
            <MenuItem>Three</MenuItem>
            <MenuDivider />
            <MenuItem>Four</MenuItem>
        </Menu>,
    );
    expect(tree).toMatchSnapshot();
});
