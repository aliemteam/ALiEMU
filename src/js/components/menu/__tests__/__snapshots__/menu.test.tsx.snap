// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render correctly 1`] = `
<div
  className="menu"
  role="menu"
>
  <div
    className="divider heading"
  >
    <h6
      className="heading"
    >
      Testing
    </h6>
  </div>
  <button
    className="menuItem"
    role="menuitem"
  >
    One
  </button>
  <button
    className="menuItem selected"
    role="menuitem"
  >
    Two
  </button>
  <button
    className="menuItem"
    role="menuitem"
  >
    Three
  </button>
  <div
    className="divider"
    role="separator"
  />
  <button
    className="menuItem"
    role="menuitem"
  >
    Four
  </button>
</div>
`;
